pyproject.toml

```diff
@@ -0,0 +1,42 @@
+[tool.poetry]
+name = "fastlane-bot"
+version = "3.1.5"
+description = "Carbon Arbitrage Bot, an open-source arbitrage protocol, allows any user to perform arbitrage between Bancor ecosystem protocols and external exchanges and redirect arbitrage profits back to the protocol."
+authors = ["Bancor Network <platon@bancor.network>"]
+license = "MIT"
+readme = "README.md"
+packages = [{include = "fastlane_bot"}]
+
+[tool.poetry.dependencies]
+python = "^3.9"
```

Comment:
thus far we said 3.8

Reply:
This is the version that was present in setup.py

Reply:
yeah. we currently test with 3.8. Maybe we should rethink this.